edweiss-app/app/(app)/lectures/slides/index.tsx

```diff
@@ -190,20 +209,22 @@ const LectureScreen: ApplicationRoute = () => {
                             </TScrollView>
 
                             <TScrollView flex={0.5} mt={15} mr={'md'} ml={'md'} mb={15}>
-
-                                {/* Dummy Questions */}
-                                {[...Array(7).keys()].map(i => renderQuestion(`showtime:dummy_question_${i}`))}
-
-                                {/* Your Question */}
-                                <FancyTextInput mb={'sm'} multiline label='Ask your questions' icon='chatbubbles-outline' placeholder='Got something on your mind? Type away!' />
+                                {/* Questions Display */}
+                                {questionsDoc?.map((question, index) => renderQuestion(question?.data.text, index))}
+
+                                {/* Enter Your Question */}
+                                <TView flexDirection='row'>
+                                    <FancyTextInput value={question} onChangeText={n => { setQuestion(n) }} mb={'sm'} multiline label='Ask your questions' icon='chatbubbles-outline' placeholder='Got something on your mind? Type away!' />
```

Comment:
In future version, could you consider using the t(``) without hardcoding strings so that we can have an easier multilanguage support.
For instance, go to `showtime.json`, add the line "ask_question": "Ask your question". don't forget to put a comma on the line above and then in the put `label=t(`showtime:ask_question`)`

Reply:
Sure that would be great, I will manage it next time